src/server/visual-diff-plugin.js

```diff
@@ -29,8 +30,7 @@ async function clearDir(updateGoldens, path) {
 	} else {
 		await Promise.all([
 			rm(join(path, PATHS.FAIL), { force: true, recursive: true }),
-			rm(join(path, PATHS.PASS), { force: true, recursive: true }),
-			rm(join(path, 'report.html'), { force: true })
+			rm(join(path, PATHS.PASS), { force: true, recursive: true })
```

Comment:
The report stuff is going to handle its own cleanup and will no longer sprinkle `report.html` files everywhere anwyay.